share/spice/geometry/geometry.js

```diff
@@ -56,8 +56,10 @@
             });
             if(svgNode.is(".fill"))
                 svgNode.css("fill", "transparent");
-            if(svgNode.is(".stroke"))
-                svgNode.css("stroke", "#000");
+            if(svgNode.is(".stroke")) {
```

Comment:
It seems like the stroke colour should only be controlled by CSS?

If you switch between the dark and light themes, the colour does not update because the inline styles take priority:

![geometry_square_at_duckduckgo](https://cloud.githubusercontent.com/assets/873785/24210912/ecd0f59a-0f00-11e7-9d95-bb62c6a1e37b.png)



We should avoid using inline styles where possible